setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+
+setup(
+    name='code-katas',
+    description="what it does",
+    version='0.1',
+    author="",
+    license='MIT',
+    py_modules=['vasya_clerk',''],
```

Comment:
You should update this to include your other modules that are holding your kata solutions